cmd/config/internal/commands/run_test.go

```diff
@@ -154,6 +154,34 @@ kind: Foo
 apiVersion: v1
 `,
 		},
+		{
+			name: "star",
+			args: []string{"run", "dir",
+				"--enable-star",
+				"--star-path", "a/b/c",
+				"--star-name", "foo",
+				"--", "Foo", "g=h"},
+			path: "dir",
+			expected: `
+metadata:
+  name: function-input
+  annotations:
+    config.kubernetes.io/function: |
+      starlark: {path: a/b/c, name: foo}
+data: {g: h}
+kind: Foo
+apiVersion: v1
+`,
+		},
+		{
+			name: "star-not-enabled",
+			args: []string{"run", "dir",
```

Comment:
Maybe add another case like this 

[]string{"run", "dir",
                                 "--image", "some_image",
				"--star-path", "a/b/c",
				"--star-name", "foo",
				"--", "Foo", "g=h"},

Reply:
done